src/common/Logo.jsx

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+
+const Logo = () => {
+  return (
```

Comment:
When your component does nothing other than return JSX code (HTML with JS), you can get rid of the curly brackets and just use parenthesis instead and loose the word return.
`const Logo = () => {
  return (
  ...
  `
  
  Becomes:
  `const Logo = () => (
  ...`